data/remnant ships.txt

```diff
@@ -144,8 +150,138 @@ ship "Starling" "Starling (Thrasher)"
 	gun
 	gun
 
+	
+ship "Gull"
+	sprite "ship/gull"
+	thumbnail "thumbnail/gull"
+	attributes
+		category "Light Freighter"
+		licenses
+			Remnant
+		cost 9500000
+		"shields" 6500
+		"hull" 2500
+		"required crew" 3
+		"bunks" 20
+		"mass" 270
+		"drag" 6
+		"heat dissipation" 0.7
+		"fuel capacity" 600
+		"ramscoop" 2
+		"cargo space" 175
+		"outfit space" 280
+		"weapon capacity" 81
+		"engine capacity" 91
+		"shield generation" 2.7
+		"shield energy" 1.8
+		"hull repair rate" 1
+		"hull energy" 0.75
+		"cloak" .02
+		"cloaking energy" 5
+		"cloaking fuel" .2
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Inhibitor Cannon" 1
+		"Point Defense Turret"
+		
+		"Epoch Cell"
+		"Crystal Capacitor"
+		"Emergency Ramscoop" 2
+		"Quantum Key Stone"
+		"Scanner Software"
+		
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+	
+	engine -12 70 0.7
+	engine 12 70 0.7
+	engine 0 78 1
+	gun 0 -96 "Inhibitor Cannon"
+	gun -7 -94
+	gun 7 -94
+	turret 0 26 "Point Defense Turret"
+	explode "tiny explosion" 40
+	explode "small explosion" 20
+	explode "medium explosion" 10
+	"final explode" "final explosion small" 1
+	description "While the starling was the first successful ship to use the technology derived from the alien ruins, the Gull is the most recent. With the recent increase in Korath making foraging for resources a riskier endeavor, the Remnant felt the need to develop a specialized freighter for hauling cargo from remote sites without revealing their presence to hostile aliens."
```

Comment:
>description "While the starling was the first successful ship to use the technology derived from the alien ruins, the Gull is the most recent. With the recent increase in Korath making foraging for resources a riskier endeavor, the Remnant felt the need to develop a specialized freighter for hauling cargo from remote sites without revealing their presence to hostile aliens."

description "While the starling was the first successful ship to use the technology derived from the alien ruins, the Gull is the most recent. With the increase in Korath activities, foraging for resources has become a risky endeavor. The Remnant had felt the need to develop a specialized freighter for hauling cargo from remote sites without revealing their presence to hostile aliens."

Reply:
Thanks. Changed.